src/ocean/util/container/cache/ExpiringCache.d

```diff
@@ -304,9 +323,6 @@ class ExpiringCache ( size_t ValueSize = 0 ) : Cache!(ValueSize, true),
     }
     body
     {
-        verify (this.lifetime > 0,
```

Comment:
Removal of this check could be in its own commit.

Reply:
It is not removed, it is moved several lines below to check both lifetimes depending on item state.

Reply:
Oh sorry, I thought you removed it because it was already checked in the constructor. Ignore my comment then.